docs/references/nextjs/auth.mdx

```diff
@@ -22,7 +55,7 @@ const { userId } : { userId: string | null } = auth();
 }
 ```
 
-## Data fetching
```

Comment:
Hash redirect? Sorry if you've already done these, just marking them all in case not